background-fetch/META.yml

```diff
@@ -4,3 +4,11 @@ links:
       results:
         - test: abort.https.window.html
           status: ERROR
+    - product: chrome
+      url: https://crbug.com/882282
```

Comment:
The issue started out as merely tracking new idlharness.js failures, presumably because of IDL changes, but yeah then it got merged with some more important things. So let's leave it then.